plugins/cj-switch.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cj-switch
```

Comment:
Comment-only: Have you considered a name like cj-toggle or cronjob-toggle or toggle-cronjob?

This is not a requirement but I think your chances of discovery would be better.

Reply:
Thank you for the hint. That's much better than switch.